src/controller/MahjongRound.ts

```diff
@@ -2,18 +2,21 @@
  * Please read the specs below for the functionality that the prototype should support.
```

Comment:
All of these enums can be autogenerated by Prisma if defined in the schema already, in fact I do this in the original project